### PR TITLE
bump: Bump of CDC to 0.27

### PR DESCRIPTION
**Pull request by AyakorK**
#### :tophat: Description
*Please describe your pull request.*
Bump of CDC to 0.27 (based on master because had some pushes that were more recents than develop)
We need to ensure that the main custo (SAML SSO) is still available and than this bump didn't got rid of it)

#### :pushpin: Related Issues
*Link your PR to an issue*
- [Cour des comptes - Montée de version 0.27](https://www.notion.so/opensourcepolitics/Cour-des-comptes-Mont-e-de-version-0-27-889baef0c142421d8f120279717b47d9?pvs=4)

#### Tasks
- [ ] Bump to 0.27.5 (compatibility issue in 0.27.6 we need to check this afterwards for every instance)
- [ ] Modify files and update them to the release/0.27-stable of decidim
- [ ] Update the CI and add the zeitwerk check
- [ ] Update the Dockerfile to ensure that it works everytime (based it from the dcd-app)
- [ ] Update the profile to add some fixes related to the tests
- [ ] Add missing locales
- [ ] Global rubocop refactor

### :camera: Screenshots
*Please add screenshots of the changes you're proposing if related to the UI*

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,83 +1,41 @@
...
-
-destroy-scw:
-	terraform -chdir=deploy/providers/scaleway destroy
-
-### Docker usage
+run: up
+	@make create-seeds
 
-# Docker images commands
+up:
```
Comment (Quentinchampenois) [inline Makefile:4]: Can you backport this PR on the bump ? 
https://github.com/OpenSourcePolitics/decidim-app/pull/554/files

**Review by Quentinchampenois — CHANGES_REQUESTED** [review]
Just a backport needed otherwise it looks great !

**Review by BarbaraOliveira13 — APPROVED** [review]
I ran It. Everything works 🎉  (sign in, sign up, proposals, reviews, & notifications...) except for one thing: I just noticed that the awesome-map displays nothing in my app locally.

.
Capture d’écran 2024-07-08 à 13 58 43
Capture d’écran 2024-07-08 à 14 01 41

**Comment by AyakorK** [comment]
Hey Barbara ! Thanks for the feedback ! 

I may be wrong but it does not display anything probably because you don't have the env var to activate the API locally, which may cause this issue.

If you think that it's not the cause and that's a real issue then don't mind telling me again ! I'll work on it afterwards !

**Review by Quentinchampenois — APPROVED** [review]

**Review by luciegrau — APPROVED** [review]

**Pull request merged**